.vale/styles/RedHat/Spelling.yml

```diff
@@ -384,9 +407,8 @@ filters:
   - Traefik
   - Uber
   - UIDs
-  - URIs?
-  - url
-  - URLs?
+  - URI
+  - URL
```

Comment:
"URLs" and "URIs" acceptable usages aren't they?

Reply:
@aireilly Just to clarify, I removed the plural instances as those are not needed. The expression caters for plural instances of the listed term.
In my tests against this PR, `URLs` and `URIs` are not flagged with a spelling warning.

Also, I did wonder why several items have a question mark against them. Is that a workaround for an issue or a tag to investigate those terms?

Reply:
@aireilly  Fixed build errors!

Reply:
the question mark is regex - it means, "one or more of the preceding character".

Reply:
@aireilly - Doh :face_in_clouds: Thanks for reminding me! I restored those items.